backend/app/views/spree/layouts/admin.html.erb

```diff
@@ -34,7 +34,7 @@
 
       <%= render :partial => 'spree/admin/shared/content_header' %>
 
-      <div class="container">
+      <div class="container<%= ' content-under-header' if (content_for?(:page_title) || content_for?(:page_actions)) %>">
```

Comment:
As we discussed IRL, move this to `.main-content` and change the conditional class name to `.has-header`. This is a lot clearer on why the `padding-top` is being added.